examples/elasticsearch/movies_metric.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+- dataFrameName: RatedMovies
+  sql:
+    SELECT userid,
+           movies.movieid
+    FROM ratings
+    JOIN movies ON ratings.movieid = movies.movieid
+    GROUP BY 1,2
+    LIMIT 10
+- dataFrameName: RatedMoviesJson
+  sql:
+    SELECT to_json(struct(userid, movieid)) AS jsonValue,
```

Comment:
is it significant it will be called jsonValue?

Reply:
No